internal/db/update.go

```diff
@@ -146,7 +146,6 @@ CREATE TABLE instances (
     migration_status_string TEXT NOT NULL,
     last_update_from_source TEXT NOT NULL,
     source_id INTEGER NOT NULL,
-    target_id INTEGER NULL,
```

Comment:
Also remove from line 36

Reply:
Ah yeah, @masnax the schema above should match this one

Reply:
Ah oops, I also didn't remove the foreign key reference from the other table either. Fixed now.